ibm_catalog.json

```diff
@@ -271,6 +261,24 @@
                 }
               }
             },
+            {
+              "key": "secrets_manager_service_plan",
+              "type": "string",
+              "default_value": "standard",
+              "description": "The pricing plan to use when provisioning a Secrets Manager instance. Possible values: `standard`, `trial`. You can create only one Trial instance of Secrets Manager per account. Before you can create a new Trial instance, you must delete the existing Trial instance and its reclamation. [Learn more](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-create-instance&interface=ui#upgrade-instance-standard).",
```

Comment:
You can remove the text: `Possible values: `standard`, `trial`` because there is a dropdown box